test/referral_chaser_test.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'test_helper'
+
+class GitHubLdapReferralChaserTestCases < GitHub::Ldap::Test
+
+  def setup
+    @mock_connection = GitHub::Ldap.new({
+        admin_user: "Joe",
+        admin_password: "passworD1",
+        port: 389
+    })
```

Comment:
Want to use a mock here too? Or use the `options` hash to pull configuration for working connections?